.yarnrc.yml

```diff
@@ -1,3 +1,12 @@
 nodeLinker: node-modules
 
 yarnPath: .yarn/releases/yarn-3.3.0.cjs
+
+logFilters:
+    # A package cannot be found in the cache for the given package and will be fetched from its remote location.
+    - code: YN0013
+      level: discard
+
+    # A package will need to be built
+    - code: YN0007
+      level: discard
```

Comment:
Any particular reason why you exclude this one, too? It shouldn't be overly spammy, right?

Reply:
imo they are fine, and we can keep them in.